readme.md

```diff
@@ -179,6 +179,7 @@ Made with Electron.
 - [Mailspring](https://github.com/Foundry376/Mailspring) - Extensible email client. (Fork of Nylas Mail)
 - [Manta](https://github.com/hql287/Manta) - Invoicing for freelancers with beautiful and customizable templates.
 - [Headset](https://github.com/headsetapp/headset-electron) - Discover, collect, and listen to music from YouTube.
+- [electerm](https://github.com/electerm/electerm) - terminal/ssh/sftp client(mac, win, linux), it works like a combination of `guake` and `xshell`.
```

Comment:
The description needs to be improved.

> it works like a combination of `guake` and `xshell`.

This is only useful info if the user knows what `guake` and `xshell` is. The description should be clear to anyone.